macosx/Torrent.mm

```diff
@@ -1996,11 +1996,10 @@ - (void)renameFinished:(BOOL)success
 
     if (success)
     {
-        using WeakUpdateNodeAndChildrenForRename = void (^__block __weak)(FileListNode*);
-        using UpdateNodeAndChildrenForRename = void (^)(FileListNode*);
-
         NSString* oldName = oldPath.lastPathComponent;
-        WeakUpdateNodeAndChildrenForRename weakUpdateNodeAndChildrenForRename;
+
+        using UpdateNodeAndChildrenForRename = void (^)(FileListNode*);
+        __weak __block UpdateNodeAndChildrenForRename weakUpdateNodeAndChildrenForRename;
```

Comment:
Yes, I've seen this one while building Xcode project in analyze mode
This is what StackOverflow answers were suggesting, LGTM